app/models/article.rb

```diff
@@ -582,7 +582,7 @@ def evaluate_markdown
 
     self.description = processed_description if description.blank?
   rescue StandardError => e
-    errors.add(:base, ErrorMessages::Clean.call(e.message))
+    errors.add("Post Content", ErrorMessages::Clean.call(e.message))
```

Comment:
I'm not sure of the implications of this or if `Post Content` is the correct verbiage to use here. But for this case it sure looks a lot better than `base:` 😅

Reply:
Well seeing as it’s communicating errors, perhaps “Error” is better? 😄

Reply:
Oh, I missed this part of the conversation because it was resolved. My [review comment](https://github.com/forem/forem/pull/16704#discussion_r820952069) asked about this change.